<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWindow</class>
 <widget class="QMainWindow" name="chatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>745</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>892</width>
    <height>745</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>892</width>
    <height>745</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fff</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="contactPanel" stretch="0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="name">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">margin:5px 5px 0px 5px;font-size:30px;color:#0d6efd;</string>
          </property>
          <property name="text">
           <string>ùêíùê≤ùêßùêúùê°ùê´ùê®ùêßùê≤</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="contactsList"/>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">margin:0px 5px 5px 5px;padding:10px 0px 10px 0px;border-radius:5px;border:1px solid #0d6efd;background-color: #0d6efd;font-weight: 400;line-height: 1.5;text-align: center;vertical-align: middle;font-size: 20px;color: #fff;</string>
          </property>
          <property name="text">
           <string>Add New Contact</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Chatpanel">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="headName">
        <property name="styleSheet">
         <string notr="true">font-size:30px;color:#fff;background-color:#333399;padding:5px 0px 5px 10px;margin-right:8px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="chatArea">
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#3366CC;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>684</width>
           <height>574</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>0</y>
            <width>681</width>
            <height>571</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="chatLayout"/>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="message">
          <property name="styleSheet">
           <string notr="true">margin:10px 0px 10px;padding:5px;font-size:17px</string>
          </property>
          <property name="placeholderText">
           <string>Enter message here...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="styleSheet">
           <string notr="true">background-color:#333399;color:#fff;font-size:20px;margin:15px 8px 15px 5px;padding:5px 15px 5px 15px;border:0px solid transparent;border-radius:5px</string>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
